sympy/sets/sets.py

```diff
@@ -635,6 +635,9 @@ def _boundary(self):
     def _measure(self):
         raise NotImplementedError("(%s)._measure" % self)
 
+    def _eval_evalf(self, prec):
+        return self.func(*[arg._evalf(prec) for arg in self.args])
```

Comment:
I would suggest this as it would keep #20379 fixed.

return self.func(*[arg.evalf(n=prec_to_dps(prec)) for arg in self.args])